### ISSUE TITLE
[CJ4] Frequency Data FMC page issues with long localizer descriptions

### ISSUE THREAD

**Issue by gr33nwich** [Working-Title-MSFS-Mods/fspackages#1158]
Using the CJ4 (0.12.1) with the latest Navigraph NAVDATA (AIRAC 2104) installed results in display issues on the `FREQUENCY DATA` page on the FMC. The screenshot below shows the Frequency page for the EDDF airport:

![CJ4_FrequencyData](https://user-images.githubusercontent.com/83493971/116783364-a5e10e80-aa8e-11eb-9a64-05bab1b4e52c.PNG)

If the set of displayed frequencies contains localizer frequencies, long descriptors such as `ILS/GS CAT III RW25C` will overlap the shorter COM frequency descriptions such as `GROUND` or `ATIS`, resulting into something unreadable like `ATILS/GS CAT III RW25C`.

I'm not sure of how the real life FMC would handle this or if Navigraph are not respecting a maximum description length for the localizer freqs, but possibly there's a way to handle the descriptions in a more readable way.

**Comment by dga711** [comment]
Hi. Yeah we don't really know how real world data would label those.
We just use what comes from data and it is that long sadly. So you have to guess quite a bit.

**Comment by gr33nwich** [comment]
Given that the real life parsing method is unknown, how about the following enhancement then: Whenever a frequency descriptor contains a string such as `CAT I` or `CAT II+III` (different combinations will need to be detected), just omit it when parsing the string for the FMC page.

Staying with my initial example of Frankfurt/Main (EDDF), both the `ILS OR LOC RWY 25C` and the `CAT II/III ILS RWY 25C` NAVDATA are sharing the same localizer (IFCW) with the exact same frequency (111.55)

--> For the `FREQUENCY DATA` FMC page, the ILS category is irrelevant and can be omitted to save display space. The key difference for the pilot are the different decision altitudes associated with CAT I and CAT II/III, resulting in different minimums settings in the PFD.

**Comment by K20017** [comment]
Yes, the categories would be omitted in the real thing, would just be "ILS + letter if applicable + runway. "

**Comment by gr33nwich** [comment]
Is there any chance to support you implementing this?

**Comment by mattnischan** [comment]
I'm not sure I follow what you mean by support, @gr33nwich.
